scripts/uninstall.sh

```diff
@@ -86,6 +86,9 @@ main() {
     sed -i '' '/^autoload -U promptinit; promptinit$/d' $ZSHRC
     sed -i '' '/^prompt spaceship$/d' $ZSHRC
     # Remove Spaceship configuration
+    # SPACESHIP_RPROMPT_ORDER and SPACESHIP_PROMPT_ORDER configuration may have multiple lines
+    # which are grouped by `(`, `)`
+    sed -i '' -E '/^SPACESHIP_R?PROMPT_ORDER=\([^)]*$/,/^[^(]*)/d' $ZSHRC
```

Comment:
`sed -i` isn't POSIX compliant and this will have trouble with `GNU sed`, See https://github.com/denysdovhan/spaceship-prompt/pull/430#issue-184513638

Reply:
@salmanulfarzy  POSIX compliant finished